### PR TITLE
git-fresh-branch: add page

### PR DESCRIPTION
**Pull request by Sadeeed**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):**

#5137

### CODE REVIEW

**Comment by Sadeeed** [comment]
**For the maintainers / reviewers**
Although I am following the command mnemonic structure, I can not find anything about in the style guide and it'd be great if someone could help me understand it fully so I don't miss any thing in the future.

Thanks

**Comment by reinhart1010** [comment]
> **For the maintainers / reviewers** Although I am following the command mnemonic structure, I can not find anything about in the style guide and it'd be great if someone could help me understand it fully so I don't miss any thing in the future.
> 
> Thanks

We originally introduce them as a workaround to document commands without [long parameter names](https://github.com/tldr-pages/tldr/blob/main/contributing-guides/style-guide.md#option-syntax), such as the ones in the Command Prompt (Windows).

File: pages/common/git-fresh-branch.md
Comment (reinhart1010) [inline pages/common/git-fresh-branch.md:7]:
```suggestion
- Create an empty local branch:
```

By the way, do not use mnemonics if it isn't linked to short command parameters (i.e. `-c`)

File: pages/common/git-fresh-branch.md
Comment (reinhart1010) [inline pages/common/git-fresh-branch.md:3]:
```suggestion
> Create an empty local branch.
```

**Review by reinhart1010 — CHANGES_REQUESTED** [review]
By the way, here are some reviews from mine.

**Review by kbdharun — APPROVED** [review]
Thanks for your contribution. LGTM after @reinhart1010's suggestions.

**Review by reinhart1010 — APPROVED** [review]
Thanks!

**Pull request merged**
